modules/mqtt/destination/mqtt-worker.c

```diff
@@ -182,6 +183,13 @@ _insert(LogThreadedDestWorker *s, LogMessage *msg)
   */
 }
 
+static gint
+_log_ssl_errors(const gchar *str, gsize len, gpointer u)
+{
+  msg_error("MQTT TLS error", evt_tag_printf("line", "%.*s", (gint) len, str));
```

Comment:
Can we use `MQTTDestinationDriver *self` here to give more info about the affected driver?

Reply:
Good idea, thanks.